core/src/main/scala/kafka/server/LegacyDynamicConfigManager.scala

```diff
@@ -43,7 +43,7 @@ object ConfigType {
 }
 
 object ConfigEntityName {
-  val Default = "<default>"
+  val Default = "<default>" // TODO remove this in favor of QuotaConfigs.DEFAULT_ENTITY_NAME
```

Comment:
In the meantime, let's initialize this variable as QuotaConfigs.DEFAULT_ENTITY_NAME